llama_hub/web/news/README.md

```diff
@@ -0,0 +1,52 @@
+# News Article Loader
+
+This loader makes use of the `newspaper3k` library to parse web page urls which have news
+articles in them.
+
+## Usage
+Pass in an array of individual page URLs:
+```python
+from llama_index import download_loader
```

Comment:
canonical way to use loaders is now `from llama_hub.web.news.base import NewsArticleReader`